config/FANTEM/FT080.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+FATEM FT080 Siren
+-->
+<Product xmlns='http://code.google.com/p/open-zwave/'>
+	<Protocol refreshonnodeinfoframe="false"/>
```

Comment:
is this really required?

Reply:
Hi Fishwaldo, nice to meet you, what do you mean that this expression Protocol refreshonnodeinfoframe="false" is required? if yes, I think it is required, and I want to know if we config like this, could you please tell me does it cause any problems?
By the way, our customers have purchased many products which are produced by Fantem, and they complained that these products can't be compatible with the gateway based OpenZwave, so could you please help us handle this imminently, thanks very much.